src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100, hidden = false) {
```

Comment:
The 'hidden' property is being initialized in the Animal constructor, but it should only be relevant for Herbivores. Consider removing it from here and handling it exclusively in the Herbivore class.